NickvisionCavalier.Shared/Controllers/PreferencesViewController.cs

```diff
@@ -481,4 +540,20 @@ public void DeleteColor(ColorType type, int index)
             ColorProfiles[ActiveProfile].BgColors.RemoveAt(index);
         }
     }
+
+    /// <summary>
+    /// Copy image from given path to Cavalier's images folder
+    /// </summary>
+    public void AddImage(string path)
+    {
+        var filename = Path.GetFileName(path);
+        var i = 0;
+        while (File.Exists($"{ConfigLoader.ConfigDir}{Path.DirectorySeparatorChar}images{Path.DirectorySeparatorChar}{filename}"))
+        {
+            filename = $"{Path.GetFileNameWithoutExtension(filename)}-{i}{Path.GetExtension(filename)}";
```

Comment:
This looks like it would produce some weird file names if there are multiple images of the same name, e.g `filename-0-1-2` rather than `filename-2`